src/entity/user/UserStreakAction.ts

```diff
@@ -5,6 +5,8 @@ export enum UserStreakActionType {
   Recover = 'recover',
 }
 
+export const streakRecoverCost = 25;
```

Comment:
@sshanzel should we record `ReputationReason` for this when subtracting streak? We can then also move this to `reputationReasonAmount` to have it standardized with other events?

Reply:
Yes! Moving there!